packages/amplication-client/src/VersionControl/PendingChangesPage.tsx

```diff
@@ -46,7 +46,7 @@ const PendingChangesPage = ({ match }: Props) => {
   );
   const { data, error } = useQuery<TData>(GET_PENDING_CHANGES, {
     variables: {
-      resourceId: resource,
+      projectId: project,
```

Comment:
Let's rename the parameter from `project` to `projectId` to be consistent. If it's not an entire Project object, then it should be named `projectId`.

Reply:
I think it is called like that because it comes from `match.params` and in the params is called "project" and not "projected"

Reply:
You are right, It won't take more than a few minutes, but if I change it in `appRoutes` I will have to change it in all the places that we use the `match.params` and this is out of my task scope.